sycl/doc/extensions/proposed/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -173,163 +169,45 @@ class annotated_arg {
 `annotated_arg` is a class template, parameterized by the type of the underlying
 allocation `T` and a list of associated properties specified by `PropertyListT`.
 
+`T` can be any type except the following types or a structure containing one of
+the following types: sycl::accessor, sycl::stream, sycl::sampler, and
+sycl::half_type.
```

Comment:
There are no such types `sycl::sampler` or `sycl::half_type`.  I think the list of prohibited types should be:

* `sycl::accessor`
* `sycl::stream`
* `sycl::local_accessor`
* `sycl::unsampled_image_accessor`
* `sycl::sampled_image_accessor`
* `sycl::half`

Reply:
I found sampler in `include/CL/sycl/sampler.hpp`. I see it's in the old header folders so maybe it is being removed. I took `half_type` from the header file, not realizing that the class name is just `half`. Thanks for the corrections! Updated.